GitUI/CommandsDialogs/FormCleanupRepository.cs

```diff
@@ -118,14 +173,51 @@ private void AddPath_Click(object sender, EventArgs e)
                 && Directory.Exists(subFoldersToClean)
                 && !subFoldersToClean.Equals(Module.WorkingDirGitDir.TrimEnd(Path.DirectorySeparatorChar)))
             {
-                checkBoxPathFilter.Checked = true;
-                textBoxPaths.Enabled = true;
-                if (textBoxPaths.Text.Length != 0)
+                checkBoxIncludePathFilter.Checked = true;
+                textBoxIncludePaths.Enabled = true;
+                if (textBoxIncludePaths.Text.Length != 0)
                 {
-                    textBoxPaths.Text += Environment.NewLine;
+                    textBoxIncludePaths.Text += Environment.NewLine;
                 }
 
-                textBoxPaths.Text += string.Join(Environment.NewLine, subFoldersToClean);
+                string userPath = string.Join(Environment.NewLine, subFoldersToClean);
+                return userPath;
+            }
+            else
+            {
+                return null;
+            }
+        }
+
+        private string? RequestUserFilePath()
+        {
+            using (OpenFileDialog openFileDialog = new())
+            {
+                openFileDialog.InitialDirectory = Module.WorkingDir;
+                openFileDialog.RestoreDirectory = true;
+
+                var result = openFileDialog.ShowDialog(this);
+
+                string fileToExclude;
+                if (result == DialogResult.OK
+                    && (fileToExclude = openFileDialog.FileName).StartsWith(Module.WorkingDir)
+                    && File.Exists(fileToExclude))
```

Comment:
Doesn't https://learn.microsoft.com/dotnet/api/system.windows.forms.openfiledialog.checkfileexists enforce this?

Reply:
Yep! Fixed it.